e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -91,45 +92,79 @@ describe("scenarios > dashboard > tabs", () => {
     });
   });
 
-  it("should allow moving dashcards between tabs", () => {
-    visitDashboardAndCreateTab({
-      dashboardId: ORDERS_DASHBOARD_ID,
-      save: false,
-    });
+  it(
+    "should allow moving dashcards between tabs",
+    { scrollBehavior: false },
+    () => {
+      visitDashboardAndCreateTab({
+        dashboardId: ORDERS_DASHBOARD_ID,
+        save: false,
+      });
 
-    goToTab("Tab 1");
+      goToTab("Tab 1");
 
-    cy.log("should stay on the same tab");
-    cy.findByRole("tab", { selected: true }).should("have.text", "Tab 1");
+      cy.log("add second card");
+      addTextBoxWhileEditing("Text card");
 
-    getDashboardCard(0).then(element => {
-      cy.wrap({
-        width: element.outerWidth(),
-        height: element.outerHeight(),
-      }).as("originalSize");
-    });
+      cy.log("should stay on the same tab");
+      cy.findByRole("tab", { selected: true }).should("have.text", "Tab 1");
 
-    cy.log("move card to second tab");
+      getDashboardCard(0).then(element => {
+        cy.wrap({
+          width: element.outerWidth(),
+          height: element.outerHeight(),
+        }).as("card1OriginalSize");
+      });
 
-    moveDashCardToTab({ tabName: "Tab 2" });
+      getDashboardCard(1).then(element => {
+        cy.wrap(element.offset()).as("card2OriginalPosition");
+      });
 
-    getDashboardCards().should("have.length", 0);
+      cy.log("move second to second tab first, then the first one");
```

Comment:
nit: improving clarity
```suggestion
      cy.log("move second card to second tab first, then the first card");
```